tests/cases/fourslash/codeFixPromiseToAsyncRej.ts

```diff
@@ -0,0 +1,25 @@
+/// <reference path='fourslash.ts' />
+
+////function f():Promise<any> {
+////    return fetch('http://yahoo.com').then(result => { console.log(result); }, rejection => { console.log("rejected:", rejection); });
+////}
+
+verify.getSuggestionDiagnostics([{
+    message: "Convert to use async and await",
+    code: 95055,
+    reportsUnnecessary: true,
+}]);
+
+verify.codeFix({
+    description: "Remove unreachable code",
+    index: 0,
+    newFileContent:
+`async function f() {
+   try{
+       var result = await fetch('http://yahoo.com);
+   }catch(rejection){
+       console.log("rejected", rejection);
+   }
+   console.log(result);

```

Comment:
If there is an exception then `result` will be undefined -- don't think the original code had that issue.